content/hi/docs/reference/glossary/replica-set.md

```diff
@@ -0,0 +1,20 @@
+---
+title: रेप्लिकासेट (ReplicaSet)
+id: replica-set
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  रेप्लिकासेट सुनिश्चित करता है कि पॉड प्रतिकृतियों की एक निर्दिष्ट संख्या एक समय में चल रही हैं।
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```

Comment:
You need to insert deployment written in the Devnagari script here. It is missing in the Glossary tooltip text section.
```suggestion
वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip text="" term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```